app/.storybook/.babelrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
I've converted this to `.js` so I could copy from the Webpack config without converting to JSON. In a separate PR we could extract this to a common config as it's now shared across Webpack, Jest and Storybook.